.pre-commit-config.yaml

```diff
@@ -129,10 +129,10 @@ repos:
     rev: v0.1.3
     hooks:
       - id: ruff
-        args: [--fix, --unsafe-fixes, --exit-non-zero-on-fix]
+        args: [--fix, --exit-non-zero-on-fix]
       - id: ruff-format
   - repo: https://github.com/OCA/pylint-odoo
-    rev: v8.0.19
+    rev: v9.0.4
```

Comment:
Maybe we do not want to change this before the release. I fixed some of the issues, but not all.